src/script/page/MainContent/panels/preferences/accountPreferences/AccountInput.tsx

```diff
@@ -92,134 +110,103 @@ const AccountInput: React.FC<AccountInputProps> = ({
     }
     setInput(value);
   };
+
   const iconUiePrefix = rest['data-uie-name'] ?? 'account-input';
+
   return (
     <div
       css={{
-        '.edit-icon': {
-          opacity: 0,
-          transition: 'opacity 0.2s ease-in-out',
-        },
-        ':hover .edit-icon': {
-          opacity: 1,
-        },
-        backgroundColor: isEditing ? 'var(--preference-account-input-bg)' : 'transparent',
         display: 'flex',
         flexDirection: 'column',
         height: 56,
-        marginBottom: 8,
-
+        marginBottom: 40,
         padding: 8,
-
-        svg: {marginLeft: 8},
-
         width: 280,
       }}
     >
-      <label
-        css={{
-          color: 'var(--foreground)',
-          fontSize: '12px',
-          fontWeight: 'normal',
-          lineHeight: '1.33',
-          marginBottom: 2,
-        }}
-        data-uie-name={labelUie}
-        htmlFor={valueUie}
-      >
-        {label}
-      </label>
-      <div
-        css={{
-          position: 'relative',
-        }}
-      >
-        <div css={{alignItems: 'center', display: 'flex', lineHeight: '1.38', position: 'absolute'}}>
-          <span
-            css={{borderBottom: readOnly || isEditing ? 'none' : '1px dashed var(--foreground)'}}
-            data-uie-name={`${iconUiePrefix}-display`}
+      {!isEditing && (
+        <>
+          <label
+            className="label preferences-label"
+            css={{
+              color: 'var(--foreground)',
```

Comment:
Is it the correct color, as this variable is a part of an old design?

Reply:
I'll check it